compose/cli/log_printer.py

```diff
@@ -13,11 +13,18 @@
 class LogPrinter(object):
     """Print logs from many containers to a single output stream."""
 
-    def __init__(self, containers, output=sys.stdout, monochrome=False, cascade_stop=False):
+    def __init__(self,
+                 containers,
+                 output=sys.stdout,
+                 monochrome=False,
+                 cascade_stop=False,
+                 log_args={'follow': False}):
```

Comment:
Using a mutable value as a default kwarg value can lead to a lot of confusion.

Instead of this default, please use

```python

def __init__(self, ..., log_args=None):
    log_args = log_args or {}
    ...
```